File: math/0263_ugly_number/0263_ugly_number.go
package main 

import (
	"fmt"
)

func isUgly(num int) bool {
	if num <= 0 {
		return false
	}

	factors := []int{2,3,5}
	for _, factor := range factors{
		for num % factor == 0 {
			num /= factor
		}
	}

	return num == 1
}

func main() {
	num := 6
	res := isUgly(num)
	fmt.Println(res)
}

File: math/0263_ugly_number/0263_ugly_number.py

class Solution(object):
    def isUgly(self, num):
        if num == 0:
            return False
        for i in [2,3,5]:
            while num % i == 0:
                num /= i
        return num == 1


num = 6
res = Solution().isUgly(num)
print(res)